src/modules/35-user-profile/navigation/UserNav/__tests__/UserNav.test.tsx

```diff
@@ -39,4 +40,46 @@ describe('User Profile Page', () => {
       )
     ).toBeTruthy()
   })
+
+  test('Logout with no return url', async () => {
+    mockLogout = jest.fn().mockReturnValue({ logoutUrl: undefined })
+    const { getByTestId } = render(
+      <TestWrapper path="/account/:accountId" pathParams={{ accountId: 'testAcc' }}>
+        <UserNav />
+      </TestWrapper>
+    )
+    const logoutBtn = getByTestId('signout-link')
+    act(() => {
+      fireEvent.click(logoutBtn)
+    })
+
+    expect(mockLogout).toBeCalled()
+    await waitFor(() =>
+      expect(getByTestId('location').innerHTML).toEqual('/redirect?returnUrl=%2F%23%2Flogin%3Faction%3Dsignout')
```

Comment:
I wonder if something like `toHaveTextContent` might be more readable.
```suggestion
      expect(getByTestId('location')).toHaveTextContent('/redirect?returnUrl=%2F%23%2Flogin%3Faction%3Dsignout')
```